core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -15,312 +15,327 @@
-  public static final String FUSE_DEBUG_ENABLE = "alluxio.fuse.debug.enabled";
-  /** Maximum number of Alluxio paths to cache for fuse conversion. */
-  public static final String FUSE_CACHED_PATHS_MAX = "alluxio.fuse.cached.paths.max";
-  public static final String FUSE_DEFAULT_MOUNTPOINT = "alluxio.fuse.mount.default";
-  public static final String FUSE_FS_ROOT = "alluxio.fuse.fs.root";
-  /** FUSE file system name. */
-  public static final String FUSE_FS_NAME = "alluxio.fuse.fs.name";
-  public static final String OSS_ACCESS_KEY = "fs.oss.accessKeyId";
-  public static final String OSS_SECRET_KEY = "fs.oss.accessKeySecret";
-  public static final String OSS_ENDPOINT_KEY = "fs.oss.endpoint";
-  public static final String S3N_ACCESS_KEY = "fs.s3n.awsAccessKeyId";
-  public static final String S3N_SECRET_KEY = "fs.s3n.awsSecretAccessKey";
-  // Not prefixed with fs, the s3a property names mirror the aws-sdk property names for ease of use
-  public static final String S3A_ACCESS_KEY = "aws.accessKeyId";
-  public static final String S3A_SECRET_KEY = "aws.secretKey";
-  public static final String GCS_ACCESS_KEY = "fs.gcs.accessKeyId";
-  public static final String GCS_SECRET_KEY = "fs.gcs.secretAccessKey";
-  public static final String SWIFT_USER_KEY = "fs.swift.user";
-  public static final String SWIFT_TENANT_KEY = "fs.swift.tenant";
-  public static final String SWIFT_API_KEY = "fs.swift.apikey";
-  public static final String SWIFT_PASSWORD_KEY = "fs.swift.password";
-  public static final String SWIFT_AUTH_URL_KEY = "fs.swift.auth.url";
-  public static final String SWIFT_AUTH_PORT_KEY = "fs.swift.auth.port";
-  public static final String SWIFT_AUTH_METHOD_KEY = "fs.swift.auth.method";
-  public static final String SWIFT_USE_PUBLIC_URI_KEY = "fs.swift.use.public.url";
-  public static final String SWIFT_SIMULATION = "fs.swift.simulation";
-  // Authentication
-  public static final String SECURITY_AUTHENTICATION_TYPE = "alluxio.security.authentication.type";
-  public static final String SECURITY_AUTHENTICATION_CUSTOM_PROVIDER =
-      "alluxio.security.authentication.custom.provider.class";
-  public static final String SECURITY_AUTHENTICATION_SOCKET_TIMEOUT_MS =
-      "alluxio.security.authentication.socket.timeout.ms";
-  public static final String SECURITY_LOGIN_USERNAME = "alluxio.security.login.username";
-  // Authorization
-  public static final String SECURITY_AUTHORIZATION_PERMISSION_ENABLED =
-      "alluxio.security.authorization.permission.enabled";
-  public static final String SECURITY_AUTHORIZATION_PERMISSION_UMASK =
-      "alluxio.security.authorization.permission.umask";
-  public static final String SECURITY_AUTHORIZATION_PERMISSION_SUPERGROUP =
-      "alluxio.security.authorization.permission.supergroup";
-  // Group Mapping
-  public static final String SECURITY_GROUP_MAPPING = "alluxio.security.group.mapping.class";
+  public static boolean isValid(String keyStr) {
```

Comment:
I think a `HashMap` could help this method as well as the other `fromString` method?